app/scripts/modules/core/src/managed/PreDeploymentRow.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './PreDeploymentRow.less';
+
+interface IPreDeploymentRowProps {
+  children?: React.ReactNode;
+}
+
+export function PreDeploymentRow({ children }: IPreDeploymentRowProps) {
```

Comment:
i'm aware this is extremely duplicative in relation to `EnvironmentRow`, and I certainly could've been clever about making `EnvironmentRow` accept different props for both use cases. It's unclear to me whether this pre-deployment step presentation is going to work out, or if we're going to totally change it up shortly after shipping, so I want to avoid ending up with a mildly brittle, unhelpful component abstraction for a use case we end up not having for long.